### PR TITLE
Point cloud component configuration bus

### PR DESCRIPTION
**Pull request by norbertprokopiuk**
I added a configuration bus that allows the configuration change of the point cloud editor component such as passing point cloud asset or changing point size.
I also moved some of the UUIDs to public API to allow other gems creation of point cloud component

### CODE REVIEW

File: Gems/Pointcloud/Code/Include/Pointcloud/PointcloudComponentConfigurationBus.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    {
+    public:
+        AZ_RTTI(PointCloudEditorComponentRequests, PointCloudEditorComponentRequestsTypeId);
+        virtual ~PointCloudEditorComponentRequests() = default;
+        virtual void SetPointcloudAsset(AZ::Data::Asset<PointcloudAsset> asset) = 0;
+        virtual void SetPointSize(float pointSize) = 0;
+    };
+
+    using PointcloudEditorComponentConfigurationBus = AZ::EBus<PointCloudEditorComponentRequests>;
+} // namespace Pointcloud
```
Comment (michalpelka) [inline Gems/Pointcloud/Code/Include/Pointcloud/PointcloudComponentConfigurationBus.h:20]:
```suggestion
} // namespace Pointcloud

```

File: Gems/Pointcloud/Code/Source/Tools/Components/PointcloudEditorComponent.cpp [hunk 2]
```diff
@@ -86,12 +86,14 @@ namespace Pointcloud
...
         AzToolsFramework::EditorEntityInfoNotificationBus::Handler::BusConnect();
         AZ::TransformNotificationBus::Handler::BusConnect(GetEntityId());
+        PointcloudEditorComponentConfigurationBus::Handler::BusConnect(GetEntityId());
     }
 
     void PointcloudEditorComponent::Deactivate()
     {
         AZ::TransformNotificationBus::Handler::BusDisconnect();
         AzToolsFramework::EditorEntityInfoNotificationBus::Handler::BusDisconnect();
+        PointcloudEditorComponentConfigurationBus::Handler::BusDisconnect();
```
Comment (michalpelka) [inline Gems/Pointcloud/Code/Source/Tools/Components/PointcloudEditorComponent.cpp:96]: Buses should be disconnected in reverse order to connection.

**Review by michalpelka — COMMENTED** [review]
Looks good, please adjust those buses

File: Gems/Pointcloud/Code/Include/Pointcloud/PointcloudComponentConfigurationBus.h (hunk 1 above)
Comment (norbertprokopiuk) [reply Gems/Pointcloud/Code/Include/Pointcloud/PointcloudComponentConfigurationBus.h]: Done

File: Gems/Pointcloud/Code/Source/Tools/Components/PointcloudEditorComponent.cpp (hunk 2 above)
Comment (norbertprokopiuk) [reply Gems/Pointcloud/Code/Source/Tools/Components/PointcloudEditorComponent.cpp]: Done

**Review by michalpelka — APPROVED** [review]

**Pull request merged**